pkg/util/azureclient/applens/mock.go

```diff
@@ -0,0 +1,369 @@
+package applens
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+// Code taken from github.com/Azure/azure-sdk-for-go/internal/mock
```

Comment:
Should we be publicizing code under `/internal`? Either this directory doesn't exist in `azure-sdk-for-go` or I don't have access to view it.

Reply:
"sdk" was left out of the path.
It should be: [github.com/Azure/azure-sdk-for-go/sdk/internal/mock](https://github.com/Azure/azure-sdk-for-go/tree/main/sdk/internal/mock)

Reply:
ah, that explains it, thank you!